src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/IntTestData.kt

```diff
@@ -23,13 +23,38 @@ class IntTestData(
     const val DEFAULT_AGENCY_ID = "MDI"
     const val DEFAULT_CREATED_USER_ID = "B_MILLS"
     val DEFAULT_DATE_TIME_OF_INCIDENT = LocalDateTime.of(2010, 11, 12, 10, 0)
+    const val DEFAULT_DATE_TIME_OF_INCIDENT_TEXT = "2010-11-12T10:00:00"
+    const val DEFAULT_HANDOVER_DEADLINE_ISO_STRING = "2010-11-15T10:00:00"
```

Comment:
I am working on deriving the ISO String from the date - but that'll be a different commit

Reply:
Cool